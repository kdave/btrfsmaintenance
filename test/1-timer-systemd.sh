#!/bin/bash

# Test if the systemd timers are enabled as per the config file

. $(dirname $(realpath "$0"))/utils

load_config "sysconfig.btrfsmaintenance.testall"
timer_setup "systemd-timer"

systemctl list-unit-files | grep btrfs
systemctl --all list-timers | grep btrfs | rev | awk '{print $1" "$2}' | rev

timer_reset "systemd-timer"
systemctl list-unit-files | grep btrfs
systemctl --all list-timers | grep btrfs | rev | awk '{print $1" "$2}' | rev

unload_config
