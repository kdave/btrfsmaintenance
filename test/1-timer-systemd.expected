btrfsmaintenance-refresh.path              disabled
btrfs-balance.service                      static
btrfs-defrag.service                       static
btrfs-scrub.service                        static
btrfs-trim.service                         static
btrfsmaintenance-refresh.service           static
btrfs-balance.timer                        enabled
btrfs-defrag.timer                         enabled
btrfs-scrub.timer                          enabled
btrfs-trim.timer                           enabled
btrfs-balance.timer btrfs-balance.service
btrfs-defrag.timer btrfs-defrag.service
btrfs-scrub.timer btrfs-scrub.service
btrfs-trim.timer btrfs-trim.service
btrfsmaintenance-refresh.path              disabled
btrfs-balance.service                      static
btrfs-defrag.service                       static
btrfs-scrub.service                        static
btrfs-trim.service                         static
btrfsmaintenance-refresh.service           static
btrfs-balance.timer                        disabled
btrfs-defrag.timer                         disabled
btrfs-scrub.timer                          disabled
btrfs-trim.timer                           disabled
